catroidBluetoothTestServer/src/org/catrobat/catroid/bluetoothtestserver/BTServer.java

```diff
@@ -42,6 +44,7 @@
 	private static final String TAG = BTServer.class.getSimpleName();
 
 	static BTServer btServer;
+	private static final Logger LOGGER = Logger.getLogger(BTServer.class.getName());
```

Comment:
Since you have to edit other code parts, please move the LOGGER to the top block where TAG is defined.